x/auth/ante/sigverify.go

```diff
@@ -258,7 +255,7 @@ func (svd *SigVerificationDecorator) AnteHandle(ctx sdk.Context, tx sdk.Tx, simu
 
 	newSigKeys := make([]string, 0, len(sigs))
 	defer func() {
-		// remove txHashCache if got an error
+		// remove txHashCash if got an error
```

Comment:
It doesn't matter, but what does this fix mean?

Reply:
Oh!
This change is fixed previous pr, but this typo seems to be re-changed because I revert previous pr. 😁
I'll fix it. Thank you.